src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+import MyDataHelps, { StepConfiguration } from "@careevolution/mydatahelps-js";
+import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
+
+export interface ConnectDeviceAccountStepContainerProps {
+    providerName?: string;
+}
+
+export default function (props: ConnectDeviceAccountStepContainerProps) {
+    const [title, setTitle] = useState<string>();
+    const [text, setText] = useState<string>();
+    const [styles, setStyles] = useState<any>({});
+    const [deviceType, setDeviceType] = useState<string>("");
+    const [providerName, setProviderName] = useState<string>(props.providerName ?? "");
+    const [connected, setConnected] = useState<boolean>();
+
+    useEffect(() => {
+        MyDataHelps.getStepConfiguration().then(function (
+            config: StepConfiguration
+        ) {
+            if (!config) return; // allows test mode to work
+
+            setTitle(config.properties.title);
+            setText(config.properties.text);
+            if (!config.properties.deviceType) {
+                throw new Error("deviceType is required");
+            }
+            if (
```

Comment:
Perhaps an enum would be better here for the deviceType?

Reply:
Or maybe we don't need to be so strict about these values? Presumably in the MDHD step implementation we will want a dropdown of the deviceType that is limited to these 3, but would be nice to be able to override that if necessary with other values since this is just used for text description and does not map directly to anything with the provider implementation.

Reply:
Good point. I've removed the check on device type. Any other issue?